README.md

```diff
@@ -4,80 +4,27 @@
 
 [![Platform](https://img.shields.io/badge/platform-iOS-lightgrey.svg)]()
 [![Devices](https://img.shields.io/badge/devices-iPhone%20%7C%20iPad-blue.svg)]()
+[![Swift version](https://img.shields.io/badge/swift-4.1-orange.svg)]()
 
 
 The Gini Vision Library provides components for capturing, reviewing and analyzing photos of invoices and remittance slips.
 
-By integrating this library into your application you can allow your users to easily take a picture of a document, review it and - by implementing the necessary callbacks - upload the document for analysis to the Gini backend.
-
-Communication with the Gini backend is not part of this library. You can either use the [Gini API SDK](https://github.com/gini/gini-sdk-ios) or implement communication with the Gini API yourself.
+By integrating this library into your application you can allow your users to easily take a picture of a document, review it and getting analysis results from the Gini backend.
```

Comment:
`... review it and get analysis results ...`

Reply:
🤦‍♂️